v-next/hardhat-utils/src/sub-process.ts

```diff
@@ -0,0 +1,27 @@
+import { spawn } from "node:child_process";
+
+/**
+ * Spawns a detached subprocess to execute a given file with optional arguments.
+ *
+ * @param absolutePathToSubProcessFile - The absolute path to the subprocess file to be executed.
+ * @param [args=[]] - Optional arguments to pass to the subprocess.
+ *
+ * The subprocess runs in a detached mode and its standard input/output is ignored.
+ * This function does not wait for the subprocess to complete and the subprocess is unreferenced
+ * to allow the parent process to exit independently.
+ */
+export function spawnDetachedSubProcess(
+  absolutePathToSubProcessFile: string,
+  args: string[] = [],
+): void {
+  const subprocess = spawn(
```

Comment:
I think we should use `execFile` instead. The reason is that `exec` and `spawn` create a new shell, so if  you have a slow `bashrc`/`zshrc`, this will be slow.

Instead, `execFile` executes a file directly, ignoring the shell.

Reply:
We should also try to get some sense of how slow this is. Not awaiting it of course, but launching it. If it's too slow maybe doing this from a worker thread is better.

Reply:
I was wrong about this. `spawn` is what we need. I doesn't initialize a shell unless you use `shell:true`